### PR TITLE
Fixes to get Pegasus builds to pass again

### PR DESCRIPTION
**Pull request by michaelfig**
* make Ledger wallet builds work again, even under Windows
* have `agoric install` account for the existence of JS packages under `golang/*`
* reduce block batch sizes for `agoric start --reset` for dapp development
* NPM packaging updates

#dapp-pegasus-branch: mfig/sdk-updates

### CODE REVIEW

File: packages/zoe/package.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
     "ses": "^0.11.0"
   },
   "files": [
+    "bundles/",
```
Comment (katelynsills) [inline packages/zoe/package.json:66]: I didn't think that we wanted to package the bundles.  Is there a reason we want to?

File: packages/cosmic-swingset/lib/ag-solo/fake-chain.js [hunk 2]
```diff
@@ -157,5 +157,8 @@ export async function connectToFakeChain(basedir, GCI, delay, inbound) {
 
   // Start the first pretend block.
   nextBlockTimeout = setTimeout(simulateBlock, maximumDelay);
-  return makeBatchedDeliver(deliver);
+
+  // Only use 100ms batches for no specified inter-block delay.
+  // This makes for more deliveries but less overall waiting time.
+  return makeBatchedDeliver(deliver, delay ? delay * 1000 : 100);
```
Comment (katelynsills) [inline packages/cosmic-swingset/lib/ag-solo/fake-chain.js]: A nitpick, but can we define delay in a line above so that this call is simplified?

**Review by katelynsills — APPROVED** [review]
A question about packaging up the Zoe bundles, but looks good to me.

File: golang/cosmos/package.json [hunk 3]
```diff
@@ -39,6 +38,18 @@
       "@agoric"
     ]
   },
+  "files": [
+    "Makefile*",
+    "app",
+    "binding.gyp.in",
```
Comment (kriskowal) [inline golang/cosmos/package.json:44]: Is this platform-dependent generated code?

**Review by kriskowal — APPROVED** [review]

File: golang/cosmos/package.json (hunk 3 above)
Comment (michaelfig) [reply golang/cosmos/package.json:44]: > Is this platform-dependent generated code?

Nope, it's a template file used by the Makefile.

File: packages/zoe/package.json (hunk 1 above)
Comment (michaelfig) [reply packages/zoe/package.json:66]: Build steps are okay when you've got a fresh Git repo and are developing the packages in question, but for NPM tarball packages, the build products are typically installed as part of the package unless they are platform-dependent (in which case they're built at `install` time).

The bundles need to be built before Zoe can be used (and they are platform-independent), so by my reasoning above, they should be put in the NPM package.

Comment (katelynsills) [reply packages/zoe/package.json:66]: Sounds good to me!

**Pull request merged**